presentations.md

```diff
@@ -0,0 +1,29 @@
+# Presentations
+
+Every **1st Saturday** of the month from **3:30 PM EAT** @nairobilug hosts a Meetup with presentations. This document seeks to guide a speaker by giving them the necessary resources they need if they want to contribute a presentations.
+Video recordings of previous meetups are in a [YouTube Playlist](https://youtube.com/playlist?list=PLWYgxwb8YFV1OS4_kWef90dYJvm730x0I)  and text based write-ups are in a [GitHub Repo](https://github.com/nairobilug/presentations).
+
+## Step One
+Have a relevant **Linux** topic you want to discuss about.
+
+**N.B** Look out for duplicates from previous presentations. Maybe you want to build on top of previous submissions.
+
+## Step Two
+Announce your interest to the group by atleast the Tuesday leading to the presentation day (Saturday).
+
+## Step Three
+Video setup .....
```

Comment:
When I presented (live presentation) we used obsstudio to record my screen, so I had to set this up.

For virtual presentations, we pre-record the presentations and present them somehow in jitsi. @BonfaceKilz could add comments on challenges he had last time with this.

Reply:
I wasn't able to figure out how to stream my video content to jitsi. It's straightforward to do from YouTube but AFAICT streaming a pre-recorded video from your local hard drive is convoluted.